code_examples/sdk_examples/src/core_features/linking/index.ts

```diff
@@ -1,5 +1,5 @@
 import { Keyring } from '@polkadot/api'
-import type { KeyringPair } from '@polkadot/keyring/types'
+import type { KeyringPair } from '@kiltprotocol/sdk-js'
```

Comment:
I guess we still want to `import *`, right? So this would become `Kilt.KeyringPair`, for consistency.

Reply:
I have updated this as per your request